### ISSUE TITLE
Do not enable logging by default

### ISSUE THREAD

**Issue by mjmare** [neo4j/neo4j-python-driver#212]
The neo4j bolt driver pollutes the console (in my IDE or Jupyter notebook with debug messages). 
 
Logging best practice for libraries dictates that the library does NOT enable al logging handler. See http://docs.python-guide.org/en/latest/writing/logging/#logging-in-a-library and https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

Otherwise: document clearly how to silence the debug logging.

**Comment by technige** [comment]
No logging handler should be enabled by default. Can you show your code and what you are seeing?

**Comment by kirsle** [comment]
I'm having the same problem. The neo4j driver logs too much INFO-level lines (which _should_ be DEBUG level, since it very verbosely outlines all the commands and responses running under the hood).

Here is a snippet:

```
2018-02-15 19:37:34,466 [neo4j.bolt] INFO - C: RUN ("MATCH (me:User {user_id: $user_id})-[r:FRIEND {status: 'confirmed'}]-(friend:User) WHERE friend.user_id is not null RETURN friend.user_id", {'user_id': 102})
2018-02-15 19:37:34,468 [neo4j.bolt] INFO - C: PULL_ALL ()
2018-02-15 19:37:34,471 [neo4j.bolt] INFO - S: SUCCESS ({'fields': ['friend.user_id'], 'result_available_after': 1})
2018-02-15 19:37:34,471 [neo4j.bolt] INFO - S: RECORD * 10
2018-02-15 19:37:34,472 [neo4j.bolt] INFO - S: SUCCESS ({'result_consumed_after': 0, 'type': 'r'})
2018-02-15 19:37:34,476 [neo4j.bolt] INFO - C: RUN ('', {'user_id': 82})
2018-02-15 19:37:34,476 [neo4j.bolt] INFO - C: PULL_ALL ()
2018-02-15 19:37:34,478 [neo4j.bolt] INFO - S: SUCCESS ({'fields': ['friend.user_id'], 'result_available_after': 0})
```

We get pages and pages of these in our logging system for a single HTTP request that does a couple of queries. It makes it hard to debug problems with other endpoints in our app, because we can add logging to try and diagnose something, but it gets washed away quickly with dozens of neo4j logs.

**Comment by kirsle** [comment]
As a workaround, I found I can get neo4j's logger and change the level to WARNING.

```python
import logging
neo4j_log = logging.getLogger("neo4j.bolt")
neo4j_log.setLevel(logging.WARNING)
```

**Comment by mjmare** [comment]
@kirsle Thx the workaround works.

**Comment by zhenlineo** [comment]
The fix will come in 1.6 https://github.com/neo4j/neo4j-python-driver/commit/123a72f2309b7debcb426d6f7e8abfd6552b2a7d
